.github/workflows/notebook-deploy.yml

```diff
@@ -24,10 +24,10 @@ jobs:
           python3 -m pip install -r requirements-pipeline.txt
 
       - name: Remove superfluous cells
-        working-directory: src/
+        working-directory: src/notebooks
         run: |
-          mkdir -p ../notebooks
-          nb-filter-cells -i adult.ipynb -o ../notebooks/adult.ipynb -t export
+          mkdir -p ../../notebooks
+          nb-filter-cells -i adult.ipynb -o ../../notebooks/adult.ipynb -t export
```

Comment:
This is fine for now, but in the longer term it would be good to crawl the `notebooks` folder rather than have to keep the workflow in sync with the bundled notebooks.

Reply:
True. Also, longer term it should be a `Makefile` that just gets called to do all these steps.

Reply:
Yes, even better.